.gitignore

```diff
@@ -8,7 +8,7 @@
 /.bundle
 
 # Ignore all logfiles and tempfiles.
-/log/*
+/log/*.log
```

Comment:
Is this needed because this rule already exists https://github.com/DFE-Digital/get-help-with-tech/blob/main/.gitignore#L38

Reply:
good spot 👍